frontend/src/theme/baseui.ts

```diff
@@ -0,0 +1,164 @@
+    font500: { ...fontStyles }, // Selected items in selectbox
+    font600: {},
+
+    LabelXSmall: { ...fontStyles },
+    LabelSmall: { ...fontStyles },
+    LabelMedium: { ...fontStyles },
+    LabelLarge: { ...fontStyles },
+    ParagraphSmall: { ...fontStyles },
+  },
+
+  colors: {
+    white: colors.white,
+    black: colors.black,
+    primary: colors.primary,
+    primaryA: colors.primary,
+    accent: transparentize(colors.primary, 0.5),
+    tagPrimarySolidBackground: colors.primary,
+    borderFocus: colors.primary,
+    contentPrimary: colors.black,
+    inputFill: colors.lightestGray,
+    inputPlaceholder: colors.darkGray,
+    inputBorder: colors.lightestGray,
+    inputFillActive: colors.lightestGray,
+    tickMarkFillDisabled: colors.lightGray,
+    tickFillDisabled: colors.gray,
+    tickMarkFill: colors.lightestGray,
+    tickFillSelected: colors.primary,
+    calendarHeaderForegroundDisabled: colors.gray40,
+    calendarDayBackgroundSelected: colors.primary,
+    calendarDayBackgroundSelectedHighlighted: colors.primary,
+    calendarDayForegroundSelected: colors.white,
+    calendarDayForegroundSelectedHighlighted: colors.white,
+    notificationInfoBackground: colors.alertInfoBackgroundColor,
+    notificationInfoText: colors.alertInfoTextColor,
+    notificationPositiveBackground: colors.alertSuccessBackgroundColor,
+    notificationPositiveText: colors.alertSuccessTextColor,
+    notificationWarningBackground: colors.alertWarningBackgroundColor,
+    notificationWarningText: colors.alertWarningTextColor,
+    notificationNegativeBackground: colors.alertErrorBackgroundColor,
+    notificationNegativeText: colors.alertErrorTextColor,
+    progressbarTrackFill: colors.lightestGray,
+  },
+}
+
+export const mainBaseUITheme = createTheme(mainThemePrimitives, themeOverrides)
+export const sidebarBaseUITheme = createTheme(mainThemePrimitives, {
+  ...themeOverrides,
+  colors: {
+    ...themeOverrides.colors,
+    // mono100 overrides
+    datepickerBackground: colors.white,
+    calendarBackground: colors.white,
+    tickFill: colors.white,
+    tickMarkFillDisabled: colors.white,
+    menuFill: colors.white,
+
+    // mono200 overrides
+    buttonDisabledFill: colors.white,
+    fileUploaderBackgroundColor: colors.white,
+    tickFillHover: colors.white,
+    inputFillDisabled: colors.white,
+    inputFillActive: colors.white,
+
+    // mono300 overrides
+    toggleTrackFillDisabled: colors.white,
+    tickFillActive: colors.white,
+    sliderTrackFillDisabled: colors.white,
+    inputBorder: colors.white,
+    inputFill: colors.white,
+    inputEnhanceFill: colors.white,
+    inputEnhancerFillDisabled: colors.white,
+
+    // mono400 overrides
+    buttonDisabledSpinnerBackground: colors.gray40,
+    toggleTrackFill: colors.gray40,
+    sliderTrackFill: colors.gray40,
+    sliderHandleInnerFill: colors.gray40,
+    sliderHandleInnerFillDisabled: colors.gray40,
+
+    progressbarTrackFill: mainTheme.colors.gray40,
```

Comment:
technically all the colors in sidebar are using mainTheme.colors but I think colors is just duplicated for both sidebar and main right now so it works but not very error proof